### ISSUE TITLE
Github Wrapped Contribution Count does not get updated.

### ISSUE THREAD

**Issue by SarthakSKumar** [avgupta456/github-trends#221]
Tried opening my Github Wrapped in a Private Window. Contribution count isn't getting updated.

**Comment by avgupta456** [comment]
Sorry I'm not sure I fully understand. Could you describe further/provide a screenshot?

**Comment by SarthakSKumar** [comment]
I checked out my github wrapped a few days ago on the github trends website. It was 1812 contributions in total on that day. I checked out the thing today again using the same URL, but the new contribution count doesn't seem to reflect. I have 1865 contributions now, but on the website it still shows 1812.

![image](https://user-images.githubusercontent.com/81763561/206767773-81379f78-211a-4204-85a3-bb4318b6cef4.png)
